neoexchange/core/frames.py

```diff
@@ -332,6 +332,14 @@ def block_status(block_id):
         logger.error("Block with id %s does not exist" % block_id)
         return False
 
+    try:
+        obj = block.body.current_name()
```

Comment:
Could we make this `obj_name` please ? "object" is already somewhat ambiguous - are we taking about the rock/snowball or the Django model instance ? Abbreviating this further to `obj` isn't helping and too many `foo=foo` in call sequences gives bad IDL flashbacks...

Reply:
sure